src/Updater.php

```diff
@@ -198,15 +249,33 @@ protected function executeConfigurationActions(array $update_definitions) {
         $new_config = NestedArray::mergeDeep($new_config, $update_actions['add']);
       }
 
-      if ($this->updateConfig($configName, $new_config, $configChange['expected_config'], $delete_keys)) {
-        $this->logInfo($this->t('Configuration @configName has been successfully updated.', ['@configName' => $configName]));
+      $result = $this->updateConfig($configName, $new_config, $configChange['expected_config'], $delete_keys, $force, $checkOnly);
+
+      if ($checkOnly) {
+        return $result;
       }
-      else {
-        $this->logWarning($this->t('Unable to update configuration for @configName.', ['@configName' => $configName]));
+
+      switch ($result) {
+        case Updater::CONFIG_APPLIED_SUCCESSFULLY:
```

Comment:
Just one quick though - I'm not sure can we somehow propagate `$result` and send it in `ConfigurationUpdateEvent`. In that case, we could save it `Update` entity and use it for some nicer representation of update result.

What do you think?